sdk/appconfiguration/app-configuration/test/index.spec.ts

```diff
@@ -669,6 +669,11 @@ describe("AppConfigurationClient", () => {
             label: i.toString()
           })
         );
+
+        if (i !== 0 && i % 10 === 0) {
+          await Promise.all(addSettingPromises);
+          addSettingPromises = [];
```

Comment:
What brought about this change?

Reply:
We were getting rate-limited doing 200 of them at once. I figured I was skirting the edge anyways so I just made it more conservative. It's the slowest test in the bunch but there is only one.